thirdeye/thirdeye-frontend/app/pods/components/rootcause-metrics/component.js

```diff
@@ -341,117 +145,141 @@ export default Component.extend({
    * }
    */
   links: computed(
-    'urns',
+    'entities',
     function() {
-      const { urns, entities } = this.getProperties('urns', 'entities');
+      const { entities } = this.getProperties('entities');
```

Comment:
You know what I'm about to comment here 😄

Reply:
what? :D

Reply:
`this.getProperties` for getting 1 property 😉